gherkin-utils/java/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.cucumber</groupId>
+  <artifactId>gherkin-utils</artifactId>
+  <version>21.0.1-SNAPSHOT</version>
```

Comment:
Please align the version number with the existing `gherkin-utils/javascript` version (and add 1 for the patch version plus a `-SNAPSHOT` suffix).